datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -462,6 +523,28 @@ macro_rules! get_null_count_values {
     }};
 }
 
+/// Convert parquet column schema to arrow field.
+/// Copy from arrow-rs
+/// TODO: consolidate code with arrow-rs
+/// TODO: change this API public in the arrow-rs
```

Comment:
Can you please file a ticket in arrow-rs describing the change needed and add a reference to the new ticket here?

Reply:
After go through the codebase, In the datafusion we just need to consider the decimal data type.

Reply:
I remove the `TODO` and don't need a ticket to track it.